common/djangoapps/entitlements/models.py

```diff
@@ -15,7 +44,8 @@ class CourseEntitlement(TimeStampedModel):
     course_uuid = models.UUIDField(help_text='UUID for the Course, not the Course Run')
     expired_at = models.DateTimeField(
         null=True,
-        help_text='The date that an entitlement expired, if NULL the entitlement has not expired.'
+        help_text='The date that an entitlement expired, if NULL the entitlement has not expired.',
+        blank=True
```

Comment:
Does blank mean something different than null to us (I know what it means to Django)?

Reply:
Blank is strictly for Django admin, it means you don't need to fill it in there if you're creating this field from there.